packages/quantic/force-app/examples/main/lwc/exampleQuanticSearchBox/exampleQuanticSearchBox.js

```diff
@@ -0,0 +1,64 @@
+import {LightningElement, track} from 'lwc';
+
+export default class ExampleQuanticSearchBox extends LightningElement {
+  @track config = {};
+  isConfigured = false;
+
+  pageTitle = 'Quantic Search Box';
+  pageDescription =
+    'The `QuanticSearchBox` component creates a search box with built-in support for query suggestions.';
+  options = [
+    {
+      attribute: 'engineId',
+      label: 'Engine id',
+      description: 'The ID of the engine instance the component registers to.',
```

Comment:
```suggestion
      description: 'Use Engine ID to connect components with each other.',
```

I think we should add the new description we use for engine ID following etienne's PR

Reply:
This description is used in the documentation page of all the Quantic component, so I would not change this in this PR.

If we wanna change, we will need to change that for all the components at the same time in a separate PR.